src/test/java/io/vertx/core/http/HttpTestBase.java

```diff
@@ -34,7 +34,7 @@
 public class HttpTestBase extends VertxTestBase {
 
   public static final String DEFAULT_HTTP_HOST = "localhost";
-  public static final int DEFAULT_HTTP_PORT = 8080;
+  public static final int DEFAULT_HTTP_PORT = Integer.parseInt(System.getProperty("port", "8080"));
   public static final String DEFAULT_HTTPS_HOST = "localhost";
   public static final int DEFAULT_HTTPS_PORT = 4043;
   public static final String DEFAULT_HTTPS_HOST_AND_PORT = "localhost:4043";
```

Comment:
likewise I think we need a `DEFAULT_HTTP_HOST_AND_PORT` that we can use in some tests

Reply:
What name should be given to the property key.